index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+  <section id="header">
```

Comment:
Good job using a semantic element such as the section. Did you know, we have a special element to put header content in ? :sparkles: 
It is called the `header` tag

- [ ] Please use this `header` in place of the `section` tag.

```html
<header id="header">
....
</header>
```